src/tsconfig-loader.ts

```diff
@@ -90,25 +90,26 @@ function resolveConfigPath(cwd: string, filename?: string): string | undefined {
   const configAbsolutePath = walkForTsConfig(cwd);
   return configAbsolutePath ? path.resolve(configAbsolutePath) : undefined;
 }
-
 export function walkForTsConfig(
   directory: string,
-  // eslint-disable-next-line no-shadow
-  existsSync: (path: string) => boolean = fs.existsSync
+  readdirSync: (path: string) => string[] = fs.readdirSync
 ): string | undefined {
-  const configPath = path.join(directory, "./tsconfig.json");
-  if (existsSync(configPath)) {
-    return configPath;
+  const files = readdirSync(directory);
+  const filesToCheck = ["tsconfig.json", "jsconfig.json"];
+  for (const fileToCheck of filesToCheck) {
+    if (files.indexOf(fileToCheck) !== -1) {
```

Comment:
Could use `files.includes` here instead

Reply:
tsconfig-paths need to support node 4, `Array.includes` is not supported in node 4, so I use indexOf instead of includes.

see:
* https://github.com/dividab/tsconfig-paths/issues/171